ci.nix

```diff
@@ -1 +1,28 @@
-{ }
+{ system ? builtins.currentSystem
+, crossSystem ? null
+, config ? {}
+, overlays ? []
+# nixpkgs-19.09-darwin as of Jan 16th 2020
+, nixpkgs ? import (builtins.fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/f69a5b2.tar.gz)
+
+# haskell.nix as of Feb 3rd 2020
+, haskell-nix ? import (builtins.fetchTarball https://github.com/input-output-hk/haskell.nix/archive/fb49ef4.tar.gz)
+
+# pkgs is nixpkgs with the haskell-nix as agument. But we'll extend haskell-nix to allow adding additional overlays and config values.
+, pkgs ? nixpkgs (haskell-nix // {
```

Comment:
All of these arguments should come from our pins, I'd prefer not to have defaults in here that we should never use.

Reply:
This is resolved I believe.